### PR TITLE
AK/MIN-178. Add offchain_worker_balancing_test

### PR DESCRIPTION
**Pull request by kurotych**
**Description:**

Also, Fix mistake in transfer_to_liquidation_pool_should_work test

**Changes in storage or API:**

Describe storage structure changes or API changes. Make sure to update the [API docs](https://minterestfinance.atlassian.net/wiki/spaces/MINTEREST/pages/134807557/Minterest+API).

**Checklist**

- [ ] Code is covered with tests
- [ ] Code is covered with benchmarks
- [ ] Fresh benchmark results included

### CODE REVIEW

File: pallets/liquidation-pools/src/tests.rs [hunk 1]
```diff
@@ -3,8 +3,56 @@
...
+	OffchainExt, TransactionPoolExt,
+};
 use sp_runtime::traits::{BadOrigin, Zero};
 
+use test_helper::offchain_ext::OffChainExtWithHooks;
+
+#[test]
+fn offchain_worker_balancing_test() {
+	// balance ratio = 0.2 for two pools. Price the same.
+	// The offchain worker must send transaction por balancing.
```
Comment (lut1k) [inline pallets/liquidation-pools/src/tests.rs:17]:
```suggestion
	// The offchain worker must send transaction for balancing.
```

**Comment by lut1k** [comment]
Previously, we were unable to use two pallet addresses in a test environment. The method `into_account()` returned the same addresses. Has this problem been fixed?

**Review by lut1k — APPROVED** [review]

File: pallets/liquidation-pools/src/tests.rs [hunk 2]
```diff
@@ -208,7 +256,7 @@ fn calculate_ideal_balance_should_work() {
 #[test]
 fn transfer_to_liquidation_pool_should_work() {
 	ExternalityBuilder::default()
-		.liquidity_pool_balance(DOT, 500_000)
+		.liquidation_pool_balance(DOT, 500_000)
```
Comment (dev-vadim) [inline pallets/liquidation-pools/src/tests.rs:259]: Could you also fix comment below?
`// Liquidity pool value: 500_000`

Comment (kurotych) [reply pallets/liquidation-pools/src/tests.rs:259]: Fixed

File: pallets/liquidation-pools/src/tests.rs (hunk 1 above)
Comment (kurotych) [reply pallets/liquidation-pools/src/tests.rs:17]: Done

**Comment by kurotych** [comment]
> Previously, we were unable to use two pallet addresses in a test environment. The method `into_account()` returned the same addresses. Has this problem been fixed?

Discussed. Just use unique 4 bytes in ModuleId to be happy

**Review by lut1k — APPROVED** [review]

**Review by Apollo50 — APPROVED** [review]

**Pull request merged**